src/main/java/core/gui/comp/entry/ColorLabelEntry.java

```diff
@@ -54,9 +61,23 @@ public ColorLabelEntry(double sortIndex, String text, Color foreground, Color ba
         number = sortIndex;
         m_clFGColor = foreground;
         m_clBGColor = background;
+        setBorderWidth(8);
```

Comment:
Should this be configurable?  Also, there should be a constant for this value.

Reply:
Good point - what about making it depending on font size. In my case (font size is 12pt), i would suggest border width = font size * 2 / 3.